tests/dtypes/test_geometry.py

```diff
@@ -135,3 +144,113 @@ def test_geometry_params_srid(
             'coordinates': 'POINT (582710 6061887)',
         }
     ]
+
+
+def test_geometry_html(rc: RawConfig):
+    context, manifest = load_manifest_and_context(rc, '''
+    d | r | b | m | property                   | type           | ref
+    example                                    |                |
+      |   |   | City                           |                |
+      |   |   |   | name                       | string         |
+      |   |   |   | coordinates                | geometry(4326) |
+    ''')
+    result = render_data(
+        context, manifest,
+        'example/City',
+        None,
+        accept='text/html',
+        data={
+            '_id': '19e4f199-93c5-40e5-b04e-a575e81ac373',
+            '_revision': 'b6197bb7-3592-4cdb-a61c-5a618f44950c',
+            'name': 'Vilnius',
+            'coordinates': 'POLYGON',
+        },
+    )
+    assert result == {
+        '_id': Cell(
+            value='19e4f199',
+            link='/example/City/19e4f199-93c5-40e5-b04e-a575e81ac373',
+            color=None,
+        ),
+        'name': Cell(
+            value='Vilnius',
+        ),
+        'coordinates': Cell(
+            value='POLYGON',
+            link=None,
+            color=None,
+        ),
+    }
+
+
+@pytest.mark.parametrize('value, cell', [
+    ({'geom_type': 'Point', 'geom_value': [582710, 6061887], 'srid': 4326},
+     Cell(value='POINT (582710 6061887)',
+          link='https://www.openstreetmap.org/?mlat=582710.0&mlon=6061887.0#map=19/582710.0/6061887.0')),
+    ({'geom_type': 'Polygon', 'geom_value': [[[502640, 6035299], [502633, 6035297], [502631, 6035305]]], 'srid': 4326},
+     Cell(value='POLYGON', link='https://www.openstreetmap.org/?mlat=502634.6666666667&mlon=6035300.333333333'
+                                '#map=19/502634.6666666667/6035300.333333333')),
+])
```

Comment:
This is quite unreadable and you are testing two different things (1) projection conversion and (2) wkt shortening. Please, could you separate these into separate tests?

Reply:
Please press „Resove conversation“ if it is resolved, or add a comment otherwise.